static/src/stylesheets/module/experiments/_survey.scss

```diff
@@ -0,0 +1,74 @@
+.impressions-survey__heading {
+  padding: .5em 1em;
+  font-size: .8em;
+  background-color: #aad7e4;
+}
+
+.impressions-survey__title {
+  display: inline-block;
+  font-family: 'Guardian Text Sans Web', 'Helvetica Neue', 'Helvetica', 'Arial', 'Lucida Grande', 'sans-serif';
+  font-weight: bold;
+  font-size: 1.8em;
+  margin-right: 1em;
+}
+
+.impressions-survey__about {
+  font-weight: normal;
+  color: inherit;
+  text-decoration: none;
+  padding-left: 1em;
+  border-left: solid #ffffff;
+}
+
+.impressions-survey__about:hover,
+.impressions-survey__about:link,
+.impressions-survey__about:active,
+.impressions-survey__about:visited {
+  text-decoration: none;
+}
+
+.impressions-survey__body {
+  color: #ffffff;
+  margin-bottom: 1em;
+  padding: 1em 1em 1em 3em;
+  font-size: .8em;
+  background-color: #005689;
```

Comment:
When you can, please use our colour variables: https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/static/src/stylesheets/pasteup/palette/_src.scss